scripts/zones/Mhaura/npcs/Laughing_Bison.lua

```diff
@@ -12,7 +12,30 @@ function onTrade(player,npc,trade)
 end;
 
 function onTrigger(player,npc)
-    player:startEvent(333);
+	-- Each boat comes every 1152 seconds/8 game hours, 4 hour offset between Selbina and Aht Urghan
+    -- Original timer: local timer = 1152 - ((os.time() - 1009810584)%1152);
+   timer = 1152 - ((os.time() - 1009810802)%1152);
```

Comment:
Make these local vars. e.g. local timer = . . .

Reply:
also the indention is off. because of tabs.